### ISSUE TITLE
cds-navigation: nav items added after initial render are visually hidden

### ISSUE THREAD

**Issue by ashleyryan** [vmware-clarity/core#155]
## Describe the bug

When a nav item is added to the Navigation component after it's initial render, the text is visually hidden with the screen reader class and not visible. 

The repro is in react, but I don't think this is react specific. 

## How to reproduce

https://stackblitz.com/edit/clarity-react-demo-bxd4di?file=src%2Findex.html,src%2FApp.tsx

1.  Notice that the nav items are created in the `useEffect` after the initial render
2.  The text isn't visible in the left nav, almost as if it's considering the nav is collapsed

## Expected behavior

The nav text is visible

## Versions

**Clarity version:**

- [x] v5.x
- [x] v6.x

**Framework:**

- [ ] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 11_

**Device:**

- Type: [e.g. MacBook]

## Additional notes

_Add any other notes about the problem here._

**Comment by bbogdanov** [comment]
https://github.com/vmware-clarity/core/issues/82 I believe it is the same issue

**Comment by ashleyryan** [comment]
Oh good catch @bbogdanov. I briefly looked at the once but didn't dig in because of my lack of angular experience.
